Invoke-ScriptInRemoteSessions.ps1

```diff
@@ -0,0 +1,91 @@
+Param (
+    [Parameter(Mandatory = $false)] [String] $ScriptFileName = "Clear-ComputeNode.ps1",
+    [Parameter(Mandatory = $false)] [String] $Addresses = "127.0.0.1",
+    [Parameter(Mandatory = $false)] [Switch] $IndividualCredentials,
+    [Parameter(Mandatory = $false, ValueFromRemainingArguments = $true)] $ArgumentsToPass
+)
+
+function New-Sessions {
+    Param (
+        [Parameter(Mandatory=$true)] [String[]] $Addresses,
+        [Parameter(Mandatory=$false)] [Int] $RetryCount = 10,
+        [Parameter(Mandatory=$false)] [Int] $Timeout = 300000
+    )
+
+    if ($IndividualCredentials -eq $false) {
+        $Credential = Get-Credential -Message "Enter common credentials for compute nodes"

```

Comment:
What do you think about passing Credential object to this script and assuming they are the same? If we don't pass Credential object, then ask the user for each one.